src/client_side_request.cc

```diff
@@ -1608,10 +1606,17 @@ ClientHttpRequest::sslBumpStart()
         return;
     }
 
+    auto establishedReply = new HttpReply;
+    establishedReply->sline.set(Http::ProtocolVersion(), Http::scOkay, "Connection established");
+    assert(!al->reply);
```

Comment:
Perhaps we can just unlock the old reply? I agree that it should not be there, but it seems like this code logic does not really depend on `al->reply` being nil here (making an assertion worth the risk).